float32 distance        # distance between two clusters
---
float32 probability     # probability of being a human